test/integration/suites/get-authorized-entries/05-assert-entities-created

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Ensure the script exits on errors
+set -e
+
+# Variables
+SPIRE_SERVER="http://localhost:8081"
+EXPECTED_SVID="spiffe://domain.test/workload"
+RETRY_COUNT=10
+RETRY_DELAY=2  # Seconds to wait between retries
+SIZE=10
+
+# Function to list entries
+list_entries() {
+    local response
+
+    response=$(curl -s -X GET "$SPIRE_SERVER/v1/entries")
+
+    echo "$response"
+}
+
+# Function to call GetAuthorizedEntries
+call_get_authorized_entries() {
+    local response
+
+    response=$(curl -s -X POST "$SPIRE_SERVER/v1/entries/authorized" -H "Content-Type: application/json" -d '{}')
+
+    echo "$response"
+}
+
+# Function to verify the GetAuthorizedEntries response
+verify_get_authorized_entries() {
+  local response=$1
+
+  for ((m=1;m<=$SIZE;m++)); do
+
+      if echo "$response" | grep -q "$EXPECTED_SVID-$m"; then
+          echo "GetAuthorizedEntries response includes expected SVID $EXPECTED_SVID."
+          return 0
+      else
+          echo "GetAuthorizedEntries response does not include expected SVID $EXPECTED_SVID."
+          echo "Response details:"
+          echo "$response"
+          return 1
+      fi
+  done
+}
+
+# Function to verify registration entry
+verify_registration_entry() {
+    local response
+
+    response=$(list_entries)
```

Comment:
why do we need to verify the full list of entries? that could be a separate integration test (if it doesn't already exist)